scripts/utils.js

```diff
@@ -176,6 +176,49 @@ async function updateDeploymentScriptVersion(nextVersion) {
   );
 }
 
+/**
+ * @param {string} importName
+ * @returns {[string, string]} [packageName, importPath]
+ */
+const getPackageNameFromImportName = (importName) => {
+  let [part1, part2, ...rest] = importName.split("/");
+
+  return importName.startsWith("@")
+    ? [`${part1}/${part2}`, rest.join("/")]
+    : [part1, [part2, ...rest].join("/")];
+};
+/**
+ * @param {string} importMapPath
+ * @param {string} nextVersion
+ */
+const updateDenoImportMap = async (importMapPath, nextVersion) => {
```

Comment:
Is this guaranteed to preserve key ordering, both for the comments and for the imports?